recipes-multimedia/aamp/aamp_git.bb

```diff
@@ -15,21 +15,27 @@ PV = "0.1.gitr${SRCPV}"
 SRC_URI = "\
```

Comment:
Can you please add BRANCH variable and make a weak assignment for SRCREV and BRANCH?

Reply:
done